sdk/core/azure-core/pom.xml

```diff
@@ -97,6 +97,12 @@
       <artifactId>netty-tcnative-boringssl-static</artifactId>
       <version>2.0.31.Final</version> <!-- {x-version-update;io.netty:netty-tcnative-boringssl-static;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-json-jackson</artifactId>
+      <version>1.1.0-beta.1</version>
```

Comment:
This is missing version tag.

Reply:
Removed as this causes a circular reference.